handlers/apps.go

```diff
@@ -85,3 +85,56 @@ func getAppMetrics(w http.ResponseWriter, r *http.Request, promSupplier promClie
 	metrics := prom.GetMetrics(&params)
 	RespondWithJSON(w, http.StatusOK, metrics)
 }
+
+// CustomMetrics is the API handler to fetch runtime metrics to be displayed, related to a single app
+func CustomMetrics(w http.ResponseWriter, r *http.Request) {
+	vars := mux.Vars(r)
+	namespace := vars["namespace"]
+	app := vars["app"]
+	template := vars["template"]
```

Comment:
basically there's _handlers_ > _business services_ > _clients_ , and `metrics` is a client and is sometimes directly manipulated from handlers, when there is no "business" code identified as such.

However here with dashboards I'm creating a dashboards service but not getting it through `business.Get()`, but building it directly instead. I don't see much value in initializing 10 services when we need only one (even if the initialization cost is cheap). At the beginning when we created `business.Get()` it was mostly because we saw in 90% of our handlers we were initializing almost the same services every time, but I would say it's not a hard rule.